src/main/java/org/reactome/addlinks/dataretrieval/ensembl/EnsemblBiomartRetriever.java

```diff
@@ -0,0 +1,232 @@
+package org.reactome.addlinks.dataretrieval.ensembl;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.reactome.addlinks.dataretrieval.FileRetriever;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
+
+public class EnsemblBiomartRetriever extends FileRetriever {
```

Comment:
Seeing as we're nitpick competing :)... you could consider capitalizing the "m" to be BioMart instead of Biomart.  If you check out their page http://ensemblgenomes.org/info/access/biomart, they capitalize the "m" in the name when referring to it in the text.

You could do a project-wide find and replace of "Biomart" for "BioMart" to do it in one swoop.

Reply:
Ooo excellent nitpick, well played

Reply:
Thank you - your serve for the match point... :).

Reply:
lol! you two are sick...
j/k ;)

Reply:
Are we @SolomonShorser-OICR or are we the only two who are truly well :)?